### DISCUSSION TITLE
CORS errors web3modal + wagmi

### DISCUSSION THREAD

**Discussion by Alim32** [WalletConnect/walletconnect-monorepo#3081] [Developer Support]
Hey

I've migrated to web3modal for wallet connect v2 however since today it suddenly started giving me CORS errors about a V1 rpc even though I've got it set to version 2 on my w3mConnector.

```
const { publicClient } = configureChains(chains, [w3mProvider({ projectId })]);
const wagmiConfig = createConfig({
    autoConnect: true,
    connectors: [
        ...w3mConnectors({ chains, version: 2, projectId }),
        new WagmiCoreConnectors.CoinbaseWalletConnector({
            chains,
            options: {
                appName: "html wagmi example",
            },
        }),
    ],
    publicClient,
});
```

The error:
Access to fetch at 'https://rpc.walletconnect.com/v1/?chainId=eip155:1&projectId=xxxxx' from origin 'https://<websitename>.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

**Comment by meri-maki** [comment]
Hi! also started getting errors all of a sudden (yesterday was working fine)
but i'm on web3js

```
/* WALLET CONNECT PROVIDER */
async function getWalletConnectProvider() {
	const provider = await EthereumProvider.init({
		projectId: mainconfig.services.walletconnect.key, // required
		chains: [1], // required
		showQrModal: true,
	})

	await provider.enable()

	return provider
}
```

<img width="583" alt="Снимок экрана 2023-07-10 в 22 57 39" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/97887955/751cce58-bd5b-41b2-b6df-f4eee4229fd8">

**Comment by Alim32** [comment]
It's suddenly gone now. Anyone have an idea what caused it?
